avian3d/src/lib.rs

```diff
@@ -255,7 +278,7 @@ fn update_proximity_sensors_system(
 
             let query_filter = SpatialQueryFilter::from_excluded_entities([owner_entity]);
             if let Some(TnuaAvian3dSensorShape(shape)) = shape {
-                let (_, owner_rotation, _) = transform.to_scale_rotation_translation();
+                let owner_rotation = transform.rotation;
```

Comment:
Why not take it directly from `rotation`?

Reply:
[`e8425b0` (#61)](https://github.com/idanarye/bevy-tnua/pull/61/commits/e8425b07984a7fd1080f135cd5b71becf98e5e2a)